DSharpPlus.Commands/Converters/DiscordChannelConverter.cs

```diff
@@ -6,55 +6,93 @@
 using DSharpPlus.Commands.Processors.SlashCommands;
 using DSharpPlus.Commands.Processors.TextCommands;
 using DSharpPlus.Entities;
-using DSharpPlus.EventArgs;
+using DSharpPlus.Exceptions;
 
 namespace DSharpPlus.Commands.Converters;
 
-public partial class DiscordChannelConverter : ISlashArgumentConverter<DiscordChannel>, ITextArgumentConverter<DiscordChannel>
+public partial class DiscordChannelConverter
+    : ISlashArgumentConverter<DiscordChannel>,
+        ITextArgumentConverter<DiscordChannel>
 {
     [GeneratedRegex(@"^<#(\d+)>$", RegexOptions.Compiled | RegexOptions.ECMAScript)]
-    private static partial Regex getChannelRegex();
+    public static partial Regex GetChannelMatchingRegex();

```

Comment:
why?

Reply:
So it may be reused in other converters/user code. Why shouldn't it be public?